src/test/java/io/pravega/connectors/flink/dynamic/table/FlinkPravegaDynamicTableITCase.java

```diff
@@ -294,6 +306,128 @@ public void testPravegaSourceSinkWithMetadata() throws Exception {
+
+        FlinkPravegaWriter<String> pravegaSink = FlinkPravegaWriter.<String>builder()
+                .forStream(stream)
+                .withPravegaConfig(SETUP_UTILS.getPravegaConfig())
+                .withSerializationSchema(serSchema)
+                .withEventRouter(event -> "fixedkey")
+                .withWriterMode(PravegaWriterMode.ATLEAST_ONCE)
+                .build();
+        content.addSink(pravegaSink).setParallelism(1);
+        env.execute();
+
+        // ---------- Produce an event time stream into Pravega -------------------
+        String sourceDDL =
+                String.format(
+                        "CREATE TABLE debezium_source (" +
+                                // test format metadata
+
+                                // + " origin_ts TIMESTAMP(3) METADATA FROM 'value.ingestion-timestamp' VIRTUAL," // unused
+                                // + " origin_table STRING METADATA FROM 'value.source.table' VIRTUAL,"
+                                "  id INT NOT NULL," +
+                                "  name STRING," +
+                                "  description STRING," +
+                                "  weight DECIMAL(10,3)" +
+                                ") WITH (" +
+                                "  'connector' = 'pravega'," +
+                                "  'controller-uri' = '%s'," +
+                                "  'scope' = '%s'," +
+                                "  'security.auth-type' = '%s'," +
+                                "  'security.auth-token' = '%s'," +
+                                "  'security.validate-hostname' = '%s'," +
+                                "  'security.trust-store' = '%s'," +
+                                "  'scan.execution.type' = '%s'," +
+                                "  'scan.streams' = '%s'," +
+                                "  'sink.stream' = '%s'," +
+                                "  'format' = 'debezium-json'" +
+                                ")",
+                        SETUP_UTILS.getControllerUri().toString(),
+                        SETUP_UTILS.getScope(),
+                        SETUP_UTILS.getAuthType(),
+                        SETUP_UTILS.getAuthToken(),
+                        SETUP_UTILS.isEnableHostNameValidation(),
+                        SETUP_UTILS.getPravegaClientTrustStore(),
+                        "streaming",
+                        stream,
+                        stream);
+        String sinkDDL =
+                "CREATE TABLE sink (" +
+                        "  name STRING," +
+                        "  weightSum DECIMAL(10,3)," +
+                        "  PRIMARY KEY (name) NOT ENFORCED" +
+                        ") WITH (" +
+                        "  'connector' = 'values'," +
+                        "  'sink-insert-only' = 'false'" +
+                        ")";
+        tEnv.executeSql(sourceDDL);
+        tEnv.executeSql(sinkDDL);
+
```

Comment:
We need a comment block to explain what are these changelogs from the source doing.